### ISSUE TITLE
[improvement] replace core/intro with a more beginner friendly tutorial

### ISSUE THREAD

**Issue by razetime** [jsoftware/j-playground#59]
[reported at the J forums](http://www.jsoftware.com/pipermail/general/2022-May/039153.html)

**Comment by rdm** [comment]
I imagine that before this can happen, someone would have to write that tutorial or lab.

But this also raises some questions about what kinds of beginners we expect -- what their interests would be, and what it is about J which would be useful or attractive to them.

**Comment by razetime** [comment]
> I imagine that before this can happen, someone would have to write that tutorial or lab.

Yeah, indeed. Or we could have an iframe that shows the J Primer.

Eitherway, for now, this is a speculative issue. It may not be entirely necessary to remove core/intro.

**Comment by JoeBognerINSIGHT** [comment]
I am understanding the request to be "provide user friendly names of the labs in the labs menu instead of the lab path"...

That seems like a reasonable request. When I added the labs menu, I did a quick and dirty way to programmatically generate the list by using the Linux find and grep commands to weed out labs that seemed incompatible with the playground at the time

Someone can go through and change the menu to be the lab title instead of the file path (similar to how they show up on jqt). That would help newcomers. I would probably recommend just doing a one time, semi tedious exercise of updating the titles in the menu

Seems like a good beginner contribution to the playground code. @razetime  do you want to take it on?

**Comment by joebo** [comment]
I am understanding the request to be "provide user friendly names of the labs in the labs menu instead of the lab path"...

That seems like a reasonable request. When I added the labs menu, I did a quick and dirty way to programmatically generate the list by using the Linux find and grep commands to weed out labs that seemed incompatible with the playground at the time

Someone can go through and change the menu to be the lab title instead of the file path (similar to how they show up on jqt). That would help newcomers. I would probably recommend just doing a one time, semi tedious exercise of updating the titles in the menu

Seems like a good beginner contribution to the playground code. @razetime  do you want to take it on?

**Comment by razetime** [comment]
Sure. I'll add the titles in.
